Documentation/ContentMapping/Index.rst

```diff
@@ -40,19 +40,40 @@ Dynamic Content Rendering in TypoScript
 
 .. highlight:: typoscript
 
-Create a new directory :file:`Configuration/TsConfig/Page/` and inside this
-directory, a new file called :file:`Page.tsconfig` with the following
-content::
+Create a file at :file:`Configuration/page.tsconfig` the following
+content:
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/page.tsconfig
 
    @import 'EXT:site_package/Configuration/TsConfig/Page/PageLayout/*.tsconfig'
 
-This file imports all files ending on `.tsconfig` from the specified folder.
-The file :file:`Page.tsconfig` could for example handle other page TsConfig
-configurations or their imports.
+This syntax imports all files ending on `.tsconfig` from the specified folder.
+
+The file :file:`Configuration/page.tsconfig` has to be loaded with an API
+function.
+
+When there is only one site in your installation or all sites share the same
+configurations we suggest to include this file globally:
+
+.. code-block:: php
+   :caption: EXT:site_package/ext_localconf.php
```

Comment:
```suggestion
   :caption: ext_localconf.php
```

We do not use the `EXT:` prefix here anywhere AFAIK.

Reply:
I am using absolute paths that look like this all over the tutorial. This makes it easier to know for unexperienced developers where to make the changes. To us it is very clear, where the file ext_localconf.php should be located to a newcommer it is not

Reply:
OK, if it's used consistently in a tutorial, we should stick to this.